README.md

```diff
@@ -1,68 +1,29 @@
-README
-======
+# ownCloud Calendar 
 
-[![Build Status](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/build.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar-rework/build-status/master)[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar-rework/?branch=master)[![Code Coverage](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar-rework/?branch=master)
+[![Build Status](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/build.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar/build-status/master)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar/?branch=master)
+[![Code Coverage](https://scrutinizer-ci.com/g/owncloud/calendar-rework/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar/?branch=master)
 
+**A calendar app for [ownCloud](https://owncloud.org).**  
 
-This is the rework of the calendar app. <br>
-It will replace the [old calendar app](https://github.com/owncloud/calendar) with ownCloud 9.0.
+It will replace the [old calendar app](https://github.com/owncloudarchive/calendar) with ownCloud 9.0, and is only the frontend for the new CalDAV-Backend which has moved into the [ownCloud Core](https://github.com/owncloud/core). 
 
-It's __in development__, don't use it in production.
+![](https://raw.githubusercontent.com/owncloud/screenshots/master/calendar/calendar.png)
 
-### Maintainers:
+## Maintainers:
 
  - [Georg Ehrke](https://github.com/georgehrke)
  - [Raghu Nayyar](https://github.com/raghunayyar)
  - [Bernhard Fröhler](https://github.com/codeling)
 
-### Joining Development
+If you’d like to join, just go through the [issue list](https://github.com/owncloud/calendar/issues) and fix some. :) We’re also in [#owncloud-calendar on freenode IRC](https://webchat.freenode.net/?channels=owncloud-calendar).
```

Comment:
please add a line break after `and fix some. :)`

(Just add two spaces and hit enter)